scripts/create-preview-tarballs.js

```diff
@@ -97,16 +97,21 @@ async function main() {
   ])
   const packages = JSON.parse(lernaListJson.stdout)
   const packagesByVersion = new Map()
+  const PR_NUMBER = process.env.GH_PR_NUMBER
+  const basePackageUrl = PR_NUMBER
+    ? `https://vercel-packages.vercel.app/next/prs/${PR_NUMBER}/`
```

Comment:
@ijjk  Depending on PR numbers for versioning is kinda bad. If you make new commits, the old builds will depend on new versions, making it hard to reproduce. Why
was this change needed for what you're trying to do?